conf/modules.config

```diff
@@ -189,6 +223,7 @@ process {
     }
 
     withName: CENTRIFUGE {
+        ext.prefix = { ${meta.id} }
```

Comment:
This is same as the prefix assigned by the module, right? Did you mean to assign a different prefix?

Reply:
See above!